_posts/2017/05/2017-05-15-EmpoweringOthers.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: post
+authors: ["Rayna Harris"]
+title: "What is the reward for empowering others?"
+date: 2017-05-14
+time: "20:00:00"
+category: ["Community","Teaching","Education","Workshops","Mentoring"]
+---
+
+What is the reward for empowering others? Following a few round-table discussions with Software Carpentry instructors, I gained some insight into what drives us to enthusiastically volunteer our time. I came to the conclusion that many of us share a common sense of accomplishment when we empower others. We enjoy empowering others through mentoring and education. Software Carpentry gives us a fountain-full of opportunities for empowering others through community developed lessons, community supported workshops, and world-wide community conversations. We start out volunteering a little when there i time, but we soon carve out more and more space in our schedule for Software Carpentry events and activities that empower us to empower others.  
```

Comment:
1. "drives us to enthusiastically volunteer our time" => "so many of us to volunteer our time so enthusiastically"
2. combine 2nd and 3rd sentences: "I came to the conclusion that many of us share a sense of accomplishment when we empower others through mentoring and education."
3. Third sentence: "Software Carpentry gives us a fountain-full of opportunities for doing this through..." (to avoid repeating "empower" again). Also, is this a metric fountain-full or an Imperial fountain-full?
4. "when there *is* time" (missing letter)
5. final sentence has two more uses of "empower"